packages/jss-plugin-camel-case/src/index.js

```diff
@@ -12,7 +12,9 @@ function convertCase(style) {
   const converted = {}
 
   for (const prop in style) {
-    converted[hyphenate(prop)] = style[prop]
+    const key = prop.startsWith('--') ? prop : hyphenate(prop)
```

Comment:
created a bench, https://esbench.com/bench/5c55f3b54cd7e6009ef620ec 
`prop[0] === '-' && prop[1] === '-'` seems to be much faster, its on a level where it probably doesn't matter but still kinda nice to have